packages/state-transition/src/signatureSets/indexedAttestation.ts

```diff
@@ -42,6 +42,9 @@ export function getAttestationsSignatureSets(
   signedBlock: allForks.SignedBeaconBlock
 ): ISignatureSet[] {
   return signedBlock.message.body.attestations.map((attestation) =>
-    getIndexedAttestationSignatureSet(state, state.epochCtx.getIndexedAttestation(attestation))
+    getIndexedAttestationSignatureSet(
+      state,
+      state.epochCtx.getIndexedAttestation(state.epochCtx.config.getForkSeq(signedBlock.message.slot), attestation)
```

Comment:
add `TODO: figure how to get attesting indices of an attestation once per block processing`